improver_tests/nbhood/use_nbhood/test_ApplyNeighbourhoodProcessingWithAMask.py

```diff
@@ -175,17 +178,45 @@ def setUp(self):
                 ],
             ]
         )
+        weights_data = np.array(
+            [
+                [
+                    [0.8, 0.7, 0.0, 0.0, 0.0],
+                    [0.7, 0.3, 0.0, 0.0, 0.0],
+                    [0.3, 0.1, 0.0, 0.0, 0.0],
+                    [0.0, 0.0, 0.0, 0.0, 0.0],
+                    [0.0, 0.0, 0.0, 0.0, 0.0],
+                ],
+                [
+                    [0.2, 0.3, 1.0, 1.0, 1.0],
+                    [0.3, 0.7, 1.0, 1.0, 1.0],
+                    [0.7, 0.9, 1.0, 1.0, 1.0],
+                    [1.0, 1.0, 1.0, 0.9, 0.5],
+                    [1.0, 1.0, 1.0, 0.6, 0.2],
+                ],
+                [
+                    [0.0, 0.0, 0.0, 0.0, 0.0],
+                    [0.0, 0.0, 0.0, 0.0, 0.0],
+                    [0.0, 0.0, 0.0, 0.0, 0.0],
+                    [0.0, 0.0, 0.0, 0.1, 0.5],
+                    [0.0, 0.0, 0.0, 0.4, 0.8],
+                ],
+            ]
+        )
         topographic_zone_points = [50, 150, 250]
         topographic_zone_bounds = [[0, 100], [100, 200], [200, 300]]
 
         mask_cubes = iris.cube.CubeList([])
-        for data, point, bounds in zip(
-            mask_data, topographic_zone_points, topographic_zone_bounds
+        weights_cubes = iris.cube.CubeList([])
+        for mdata, wdata, pt, bnd in zip(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable name "pt" doesn't conform to snake_case naming style](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5415593)